### DISCUSSION TITLE
How to select Vue language

### DISCUSSION THREAD

**Discussion by DotNeter-Hpf** [massCodeIO/massCode#177] [Q&A] [answered]
The default language is not found; By the way, how to set Chinese in version 2.10.0

**Comment by antonreshetov** [comment, marked as answer]
- You can use HTML lang for Vue, because it's valid HTML.
- Preferences -> Language

**Reply by DotNeter-Hpf** [reply]
Thank you very much

**Reply by antonreshetov** [reply]
@DotNeter-Hpf  Added Vue support in v3
![image](https://user-images.githubusercontent.com/17480987/183037257-1de27874-dcbb-49ba-85a8-20abf1a8e16f.png)
